src/main/java/core/basesyntax/FileEditor.java

```diff
@@ -0,0 +1,279 @@
+                        System.out.println("Размер в байтах:     " + sizeInBytes);
+                    } catch (IOException e) {
+                        e.printStackTrace();
+                    }
+                } else {
+                    System.out.println("Указанный файл не существует");
+                }
+            } else {
+                System.out.println("Указанный путь не существует");
+            }
+        }
+    }
+
+    private static void readFile(String consoleLine) {
+        consoleLine = consoleLine.trim();
+        int commandSeparator = consoleLine.indexOf(" ");
+
+        if (commandSeparator == -1) {
+            System.out.println("Не верный формат команды");
+        } else {
+            Path fullPath = Paths.get(consoleLine.substring(0, commandSeparator) + "\\"
+                    + consoleLine.substring(commandSeparator).trim());
+            Path path = fullPath.getParent();
+
+            if (Files.exists(path)) {
+                if (Files.exists(fullPath)) {
+                    List<String> lines = null;
+                    try {
+                        lines = Files.readAllLines(fullPath);
+                        if (lines.size() == 0) {
+                            System.out.println("Файл пуст");
+                        } else {
+                            for (String string: lines) {
+                                System.out.println(string);
+                            }
+                        }
+                    } catch (IOException e) {
+                        e.printStackTrace();
+                    }
+                } else {
+                    System.out.println("Указанный файл не существует");
+                }
+            } else {
+                System.out.println("Указанный путь не существует");
+            }
+        }
+    }
+
+    private static void createFile(String consoleLine, String fileData) {
+        consoleLine = consoleLine.trim();
+        int commandSeparator = consoleLine.indexOf(" ");
+
+        if (commandSeparator == -1) {
+            System.out.println("Не верный формат команды");
+        } else {
+            Path fullPath = Paths.get(consoleLine.substring(0, commandSeparator) + "\\"
+                    + consoleLine.substring(commandSeparator).trim());
+            Path path = fullPath.getParent();
+            Path fileName = fullPath.getFileName();
+
+            if (Files.notExists(path)) {
+                int answer = getAnswerYesNo("Путь \"" + path + "\" не существует. Создать?");
+
+                if (answer == 1) {
+                    try {
+                        Files.createDirectory(path);
+                    } catch (IOException e) {
+                        System.out.println("Проверьте формат пути. "
+                                + "Конечный каталог/файл не был создан");
+                    }
+                }
+            }
+
+            if (Files.exists(path)) {
```

Comment:
divide it into two methods: one for situation when file does exist and other when it doesn't